### PR TITLE
Themeable CustomButton component. Added helper functions

### PR DESCRIPTION
**Pull request by rekesan**

### CODE REVIEW

File: app/_layout.tsx [hunk 1]
```diff
@@ -3,15 +3,15 @@ import { makeTheme, DripsyProvider } from "dripsy";
...
-    $fontNoBg: "#1D2B33",
-    $bg: "#f2f2f2",
-    $border: "#A5ADB3",
-    $tint: "#404040",
-    $pageBg: '#c9ccd1'
+    $primary: "#1a74e4", // blue
+    $secondary: "#1974e5", // blue
+    $light: "#e0e0e4", // lighter gray 1
+    $font: "#475a69", // gray
+    $fontNoBg: "#1D2B33", // dark blue gray
```
Comment (jendan-nikolai) [inline app/_layout.tsx:10]: be specific in naming variables

File: app/Login.tsx [hunk 2]
```diff
@@ -143,33 +106,35 @@ const Login = () => {
...
-            buttonStyle={forgotStyle}
-            textStyle={textBtnStyleNoBg}
-            activeOpacity={0.5}
-            onPress={() => Alert.alert("Forgot password", "Clicked")}
+
+          <CustomButton
+            text="Forgot password?"
+            variant={["borderless", "width-90"]}
+            textVariant={["darkFont"]}
+            onPress={() => {
```
Comment (jendan-nikolai) [inline app/Login.tsx:124]: Is this necessary?

File: app/Login.tsx [hunk 3]
```diff
@@ -143,33 +106,35 @@ const Login = () => {
...
-            label="Create new account"
-            buttonStyle={newAccStyle}
-            textStyle={textBtnStyleNoBg}
-            activeOpacity={0.5}
-            onPress={() => Alert.alert("Create new account", "clicked")}
+          <CustomButton
+            text="Create new account"
+            variant={["outline", "width-90"]}
+            textVariant={["darkFont"]}
+            onPress={() => alert("Create new account")}
```
Comment (jendan-nikolai) [inline app/Login.tsx:135]: Is this also necessary?

File: components/Post.tsx [hunk 4]
```diff
@@ -62,75 +62,71 @@ const ImageStyled = styled(Image)({
...
             </DatePostedText>
           </NameDateView>
         </UserView>
 
-        <Button
-          icon={<Entypo name="dots-three-horizontal" size={24} color="black" />}
-          buttonStyle={IconStyle}
+        <CustomButton
+          icon={<Entypo name="dots-three-horizontal" size={16} color="black" />}
           onPress={() => alert("Clicked 3dots button")}
```
Comment (jendan-nikolai) [inline components/Post.tsx:97]: Is this necessary?

File: app/Login.tsx (hunk 3 above)
Comment (rekesan) [reply app/Login.tsx:135]: yes

File: components/Post.tsx (hunk 4 above)
Comment (rekesan) [reply components/Post.tsx:97]: yes

File: app/Login.tsx (hunk 2 above)
Comment (rekesan) [reply app/Login.tsx:124]: yes

File: app/_layout.tsx (hunk 1 above)
Comment (rekesan) [reply app/_layout.tsx:10]: refactoring the name of the colors will not affect the components that uses them. so i left it as such.

Comment (rekesan) [reply app/_layout.tsx:10]: what i mean is the F2 refactor variable name feature of vscode

**Review by jendan-nikolai — APPROVED** [review]

**Pull request closed without merging**